lib/solarwinds_apm/api/custom_instrumentation.rb

```diff
@@ -14,6 +14,37 @@ def self.included(base)
       end
 
       module ClassMethods
+        # Helper function to instrument custom function
```

Comment:
Can we be consistent and just use "method" instead of sometimes "function"? Please check for all instances in this docstring.

Reply:
Updated to use `method`